tatarin/question_handler.py

```diff
@@ -13,7 +13,7 @@ def _is_question(data):
     msg = data['text']
     msg_lower = msg.lower().rstrip()
 
-    if not msg_lower.endswith('?'):
+    if not (msg_lower.endswith('?') or '? ' in msg_lower):
```

Comment:
Even with current implementation there is no check whether the message contains anything besides question mark at the end. Although I agree that this might be a reasonable feature, this was not a point of this PR. Please let me know if you think that the proposed changes make it more likely for a user to submit meaningless questions.